backend/src/test/java/com/redhat/cloud/notifications/CrudTestHelpers.java

```diff
@@ -126,8 +125,7 @@ public static void updateBundle(Header identity, String bundleId, Bundle bundle,
                 .body(Json.encode(bundle))
                 .put("/internal/bundles/{bundleId}")
                 .then()
-                .statusCode(expectedStatusCode)
-                .contentType(familyOf(expectedStatusCode) == SUCCESSFUL ? containsString(TEXT.toString()) : any(String.class));
```

Comment:
Some REST responses no longer return a content type with Quarkus 3 while they did with Quarkus 2. As a consequence, many tests no longer check the content type.